code/modules/jobs/job_types/shaft_miner.dm

```diff
@@ -1,3 +1,35 @@
+GLOBAL_LIST_INIT(miner_callouts, list(
+	/obj/item/stack/sheet/mineral/gold = list(":H We're rich!"),
+	/obj/item/stack/ore/gold = list(":H Gold!", ":H There is gold here!", ":H There is gold!", ":H I found a gold vein!"),
+	/obj/structure/flora/ash/leaf_shroom = list(":H Mushroom!"),
+	/obj/structure/flora/ash/cap_shroom = list(":H Mushroom!"),
+	/obj/structure/flora/ash/stem_shroom = list(":H Mushroom!"),
+	/obj/structure/closet/crate/secure/loot = list(":H Abandoned crate here!", ":H Found a crate with a codelock over here!"),
+	/obj/structure/closet/crate/necropolis = list(":H Spooky chest here, someone got a key?", ":H Necropolis chest here!"),
+	/obj/structure/spawner/lavaland = list(":H Tendril spotted!", ":H Got a tendril here!", ":H Found a tendril!"),
+	/obj/structure/geyser = list(":H Got a geyser here, those chemists'll be pleased!", ":H Found a geyser!", ":H Geyser here!"),
+	/mob/living/simple_animal/hostile/asteroid/basilisk = list(":H Basilisk, watch out!", ":H Spotted a Basilisk!", ":H Look out for that Basilisk!"),
+	/mob/living/simple_animal/hostile/asteroid/brimdemon = list(":H Brimdemon, don't stand around, move!", ":H There's a Brimdemon!", ":H Watch out for the Brimdemon!"),
+	/mob/living/simple_animal/hostile/asteroid/goldgrub = list(":H Goldgrub here!", ":H Spotted a Goldgrub!", ":H Lootbug here!"),
+	/mob/living/simple_animal/hostile/asteroid/goliath = list(":H Watch out for that Goliath!", ":H Found a Goliath!", ":H There's a goliath here, don't get hit by the tendrils!"),
+	/mob/living/simple_animal/hostile/asteroid/gutlunch = list(":H Gutlunch here!", ":H Found a Gutlunch!", ":H Got a Gutlunch here!"),
+	/mob/living/simple_animal/hostile/asteroid/hivelord = list(":H Hivelord here!", ":H Spotted a Hivelord!", ":H Hivelord!"),
```

Comment:
Similar to the Basilisk/Watcher comment, shouldn't this be for legions (the not megafauna kind)? `...asteroid/hivelord/legion`